go/libkb/secret_store_osx.go

```diff
@@ -23,69 +19,31 @@ func (k KeychainSecretStore) getServiceName() string {
 }
 
 func (k KeychainSecretStore) StoreSecret(secret []byte) (err error) {
-	k.G().Log.Debug("+ StoreSecret(%s, %d)", k.accountName, len(secret))
-	defer func() {
-		k.G().Log.Debug("- StoreSecret -> %s", ErrToOk(err))
-	}()
-
-	// base64-encode to make it easy to work with Keychain Access.
-	encodedSecret := base64.StdEncoding.EncodeToString(secret)
-	attributes := kc.GenericPasswordAttributes{
-		ServiceName: k.getServiceName(),
-		AccountName: k.accountName.String(),
-		Password:    []byte(encodedSecret),
-	}
-	err = kc.RemoveAndAddGenericPassword(&attributes)
-	return
+	item := keychain.NewGenericPassword(k.getServiceName(), k.accountName, "", secret, "")
```

Comment:
Yeah added encoding (also for iOS) so it's consistent.

I was under the impression that the keychain was storing the password (utf8), but I guess it's something else?